src/zoid/qr-code/component.jsx

```diff
@@ -73,6 +73,9 @@ export function getQRCodeComponent() : QRCodeComponent {
                 onEscapePath: {
                     type: 'function'
                 },
+                onSubmitFeedback: {
```

Comment:
@andregardi - let's be explicit in setting `required` to `true` or `false`.

Reply:
Sure @remotevision !
For the moment I am keeping `required` as `false` for that.
I just tried making it true, and it does not play well with older versions of smart-payment-buttons.

Reply:
@andregardi But it is set to `true` currently.  Can you set it to `false`?

Reply:
Sorry for that!
I fixed it now.

Reply:
Not sure why we need `onSubmitFeedback`? Can't we just close the modal, does the parent need to care about whether feedback was sent or not?

Reply:
We don't need this either so you can remove.

Reply:
That is great, I will handle this event entirely on the component side.